src/app/cake/cake.class.ts

```diff
@@ -0,0 +1,13 @@
+
+
+export type CakeStatus =  'Available' |'LastUnits' |'OutOfStock';
+
+export interface Cake{
```

Comment:
As a good practice, you can add the letter I before the name of your interfaces, to identify them versus the classes. Ex. IBread or `iBread`.

Reply:
I like to avoid prefix or suffix in the interfaces, coming from a heavy dependency injection background, i prefer to just call them as they are and the implementation should describe themselves.

For example
```
MongoBread implements Bread{...}
// Instead of
Bread implements IBread{}
```
At the end you will be refering just to interface, you shouldn't be caring about the implementation.

But yeah, that's more like a team convention agreement